File: FitnessCentar/FitnessCenatr.Model/Novosti.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FitnessCentar.Model
{
    public class Novosti
    {
        public int Id { get; set; }

        public string? Naslov { get; set; }

        public string? Tekst { get; set; }

        public DateTime? DatumObjave { get; set; }

        public int? AutorId { get; set; }

      
    }
}

File: UI/fitness_admin/lib/models/novosti.dart
import 'package:json_annotation/json_annotation.dart';

part 'novosti.g.dart';

@JsonSerializable()
class Novosti{
       int? id;
       String? naslov;
       String? tekst;
       DateTime? datumObjave;
       int? autorId;

         Novosti(this.id,this.naslov,this.tekst,this.datumObjave,this.autorId);

  factory Novosti.fromJson(Map<String, dynamic> json) =>
      _$NovostiFromJson(json);

  Map<String, dynamic> toJson() => _$NovostiToJson(this);
}

